flood_adapt/object_model/benefit.py

```diff
@@ -498,29 +548,60 @@ def _make_html(self, cba):
         fig.write_html(html)
 
     @staticmethod
-    def load_file(filepath: Union[str, os.PathLike]):
-        """create Benefit object from toml file"""
+    def load_file(filepath: Union[str, os.PathLike]) -> IBenefit:
+        """Create a Benefit object from a toml file
+
+        Parameters
+        ----------
+        filepath : Union[str, os.PathLike]
+            path to a toml file holding the attributes of a Benefit object
+
+        Returns
+        -------
+        IBenefit
+            a Benefit object
+        """
 
         obj = Benefit()
         with open(filepath, mode="rb") as fp:
             toml = tomli.load(fp)
         obj.attrs = BenefitModel.parse_obj(toml)
```

Comment:
parse_object throws an error in the file on my local maschine. Not sure if there's something missing